device-to-group-command.py

```diff
@@ -0,0 +1,404 @@
+'''
+esper app usage
+
+usage: esper_group_actions [-h]
+                            -c  {uninstall, whitelist, brightness, alarm_volume, ring_volume, music_volume, notification_volume, bluetooth, wifi, gps, ping, reboot}
+                            -g  GROUP_ID
+                            -v  VALUE
+
+example:
+
+./esper_group_actions -g 52ecfb3c-d1ad-4e66-8cf9-85daff8d7f3c -c ring_volume -v 50
+./esper_group_actions -g 52ecfb3c-d1ad-4e66-8cf9-85daff8d7f3c -c ping
+
+Runs on all active devices in a group. API key is required.
+'''
+
+import sys
+import requests
+import esperclient
+import time
+from esperclient.rest import ApiException
+
+import argparse
+active_device_list = []
+inactive_device_list = []
+
+#### TYRELL CORP CONFIG #######
+
+configuration = esperclient.Configuration()
+configuration.host = 'https://tyrellcorp-api.shoonyacloud.com/api'
+configuration.api_key['Authorization'] = 'RaCh1Ty3L181AdE4uN32e593rD3ck36'
+configuration.api_key_prefix['Authorization'] = 'Bearer'
+enterprise_id = '5da4cf47-398c-4cea-82be-2796a3f23c3c'
```

Comment:
Are these real credentials ? Don't put any credentials here.

Reply:
the endpoint name is real. But the enterprise ID and api key - no. See if you can decode the api key. But I think we should just < > and say what to put there instead. better for sample code use.

Reply:
Done